android/assets/jsons/translations/French.properties

```diff
@@ -2296,7 +2296,7 @@ in cities with a [buildingFilter] = dans les villes avec [buildingFilter]
 in cities without a [buildingFilter] = dans les villes sans [buildingFilter]
 in cities with at least [amount] [populationFilter] = dans les villes avec au moins [amount] en [populationFilter]
  # Requires translation!
-in cities with [amount] [populationFilter] = 
+in cities with [amount] [populationFilter] = dans les villes avec [amount] en [populationFilter]
```

Comment:
Because in french, it means 15 "of" something.
"`dans les villes avec 15 Population`" sounds weird in french, as Population is a "global" word, not something that can be numbered (if it was "Residents" or "Inhabitants" or "Citizens", it would be OK though).

But this does not apply to Culture, Research, etc., as it is implied 15 "pts of" Research.
Example:
`Provides [amountOfCulture] culture at 30 Influence = Fournit [amountOfCulture] Culture à 30 Influence`